src/njoy21/input/PLOTR/Card9/test/Card9.test.cpp

```diff
@@ -0,0 +1,178 @@
+#define CATCH_CONFIG_MAIN
+
+#include "catch.hpp"
+
+#include "njoy21.hpp"
+
+using namespace njoy::njoy21::input;
+
+SCENARIO( "Validating card9 inputs",
+          "[PLOTR], [Card9]" ){
+
+  GIVEN( "valid PLOTR Card9 inputs" ){
+    WHEN( "All values are given" ){
+      std::vector<int> validIcon{-80,-10,0,10,80};
+      std::vector<int> validIsym(19);
+      std::iota(validIsym.begin(), validIsym.end(), 0);
+      std::vector<int> validIdash{0,1,2,3,4};
+      std::vector<int> validIccol{0,1,2,3,4,5,6,7,8};
+      std::vector<int> validIthick{0,1};
+      std::vector<int> validIshade(81);
+      std::iota(validIshade.begin(), validIshade.end(), 0);
+
+      for( auto icon : validIcon ){
```

Comment:
Very nice combination of tests. Note this takes 34.85 seconds; 73% of the total run time of the tests. This is *not* a condemnation of this set of tests; just a remark.